arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/stream/Chunk.kt

```diff
@@ -143,7 +143,7 @@ abstract class Chunk<out O> {
   open fun take(n: Int): Chunk<O> =
     when {
       n <= 0 -> empty()
-      n == 1 -> fromNullable(firstOrNull())
+      n == 1 -> just(get(0))
```

Comment:
one comment lost in the discussion:

The n <= 0 will cover against wrong params, but if there's an empty Chunk and we request 1, won't this crash here?

Reply:
Yes... that means our `Arb` is not complete. I think we need to add `Chunk.empty` as an edgecases.
I'll move this to a separate PR.